subsys/Makefile

```diff
@@ -8,3 +8,4 @@ obj-$(CONFIG_DISK_ACCESS) += disk/
 obj-$(CONFIG_CPLUSPLUS) += cpp/
 obj-y += logging/
 obj-y += debug/
+obj-$(CONFIG_MCUBOOT_IMG_MANAGER) += dfu/
```

Comment:
I changed this. I added menuconfig at top level:
`      DFU options  --->    `
and under it I putted follow configs:
`  [*] DFU image manager                                            `
`            Image manager (Image manager for mcuboot)  --->             `
`      (512) Image writer buffer size                                    `
`      (0) Mcuboot image manager Log level  `